sw/device/tests/uart_parity_break_test.c

```diff
@@ -162,6 +206,13 @@ static status_t execute_test(void) {
   LOG_INFO("Receiving data with wrong parity");
   IBEX_SPIN_FOR(uart_irq_rx_parity_err_fired, 1 * 1000 * 1000);
 
+  // Wait for host to sync and then expect to receive a line break.
+  uart_irq_rx_break_err_expected = true;
+  OTTF_WAIT_FOR(test_phase == kTestPhaseBreakErr, kCommandTimeoutMicros);
+
+  LOG_INFO("Waiting for line break");
+  IBEX_SPIN_FOR(uart_irq_rx_break_err_fired, 1 * 1000 * 1000);
```

Comment:
I would be wiser to use `ATOMIC_WAIT_FOR_INTERRUPT`

Reply:
Thanks, I'll remember this one